### ISSUE TITLE
Unable to resolve classname: :default when using (try ... (catch ... ))

### ISSUE THREAD

**Issue by adam-james-v** [borkdude/scittle#14]
When using scittle, the try/catch is failing with various different classes. js/Object, js/Error, :default were all tried with the same 'Unable to resolve classname' error. This occurs during analysis phase.

Here is a minimal page using scittle that causes this error:

```html
<html>
  <head>
    <script src="https://cdn.jsdelivr.net/gh/borkdude/scittle@0.0.1/js/scittle.js" type="application/javascript"></script>
    <script type="application/x-scittle">
      (try
        (/ 1 0)
        (catch :default e (.log js/console e)))
    </script>
  </head>
  <body>
  </body>
</html>
```

**Comment by borkdude** [comment]
Thanks, found the issue: https://github.com/borkdude/sci/issues/583

**Comment by borkdude** [comment]
Fixed in v0.0.2:

<img width="1152" alt="Screenshot 2021-07-03 at 13 08 05" src="https://user-images.githubusercontent.com/284934/124352301-df093c80-dbff-11eb-9945-58df3e215f2c.png">
